getPricesV2.js

```diff
@@ -0,0 +1,79 @@
+exports.handler = (event, context, callback) => {
+    // Setup
+    var AWS = require("aws-sdk");
+    var docClient = new AWS.DynamoDB.DocumentClient();
+ 
+    // User Inputs. 
+    var latitude = event.latitude;
+    var longitude = event.longitude;
+    var distance = event.distance;
```

Comment:
Shouldn't we be accessing the parameters using the `event.queryStringParameters.key1` kind of syntax?

As we're accepting HTTP POST requests from the client.